### DISCUSSION TITLE
part-ii/2017-87/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#390] [Question comments]
Part II 2017 Logic and Set Theory: Paper 2, Section II, H 

https://questions.tripos.org/part-ii/2017-87/

**Comment by artie2000** [comment]
<details>
<summary>(i)</summary>

$\omega+1\neq1+\omega$

</details>
<details>
<summary>(ii)</summary>

$(\omega+1)\omega\neq\omega^2+\omega$

</details>
<details>
<summary>(iii)</summary>

Yes: use synthetic definitions.

</details>
<details>
<summary>(iv)</summary>

Yes: directly prove.

</details>
<details>
<summary>(v)</summary>

No: suppose $\alpha\beta>\beta\alpha$. Then

$\alpha\alpha\beta\beta\geq\alpha\beta\alpha\beta\geq\beta\alpha\alpha\beta>\beta\alpha\beta\alpha\geq\beta\beta\alpha\alpha$.

(ew)

</details>
